earth_enterprise/src/fusion/autoingest/Asset.h

```diff
@@ -36,7 +37,7 @@
  ***  ... = asset->config.layers.size();
  ***
  ******************************************************************************/
-class AssetImpl : public khRefCounter, public AssetStorage {
+class AssetImpl : public khMTRefCounter, public AssetStorage {
```

Comment:
I'm not positive if this change was necessary or not but I switched to making all AssetImpl objects use the multi-thread ref counter to be safe.